bindings/imap-codec-python/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "imap-codec-python"
+description = "Python bindings for imap-codec"
+version = "2.0.0-alpha.1"
```

Comment:
@duesee I gave this some thought since yesterday. Maybe, it is not a good idea to use the same numbering scheme for the binding packages. Otherwise, every breaking change in the bindings would force us to bump the major version number for all crates.
At some point, when the Python bindings are really nice, this might make sense, but currently, it does not feel right to me. What do you think?

Reply:
Good point. I agree. I think it could be separated.

Reply:
Switching to `0.1.0-dev1` for now. Then we can re-adress the version to use when publishing the first version to PyPI.